CTFd/themes/admin/assets/js/pages/configs.js

```diff
@@ -124,6 +124,44 @@ function updateConfigs(event) {
   });
 }
 
+function switchMode() {
+  let new_mode = $("#user_mode option:selected").val();
+  let body = "";
+  if (new_mode == "teams") {
+    body =
+      "Going from user mode to teams mode will not change anything, but are you sure you want to make the change?";
+  } else {
+    body =
+      "Going from teams mode to user mode <b style='color:red'>WILL REMOVE ALL SUBMISSION DATA</b>. Are you sure you want to make the change?";
```

Comment:
There's no other usage of bold red text. Better to just use `<strong>`. 

I would say add two line breaks between the sentences and use "this change" instead of "the change".